service-deployment/bootstrap/grafana/grafana-configuration/pai-jobview-dashboard.json.template

```diff
@@ -281,23 +281,23 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "avg by (container_label_PAI_JOB_NAME) (rate(container_network_receive_bytes_total{container_label_PAI_JOB_NAME=~\"$job\"}[5m])) ",
```

Comment:
>[5m] [](start = 144, length = 4)

Why removes 5m time interval config?

Reply:
Once use rate function need [5m] like interval. Currently don't use that